src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkCustomCommandLine.java

```diff
@@ -1423,8 +1432,13 @@ void addPreprocessedArg(PreprocessedArg arg) {
         numArgs += arg.numArgs();
       }
 
-      void addString(String arg) {
```

Comment:
Can you keep this method? Some callers still have a `String` reference, for which it helps performance to avoid the type check, and enhances readability that it's definitely a string.